src/pages/about.vue

```diff
@@ -1,13 +1,54 @@
 <template>
-    <div class="page-text">
-        <p v-if="appConfig" class="ma-4 text-body-1">
-            {{ appConfig.title }} is in development. Check back later for more about the application.
-        </p>
-    </div>
+    <v-container class="page-text" style="min-height: 100vh">
+        <v-col>
+            <v-row>
+                <h1 class="text-h4">About this site</h1>
+                <p class="text-body-1">
+                    This is an interactive map tool developed to allow users to access and explore high resolution
+                    dengue transmission intensity in countries where dengue is endemic. These maps were generated by
+                    fitting environmentally driven geospatial models to force of infection estimates derived from
+                    surveillance data including case-notification data and cross-sectional serological surveys. We
+                    provide estimates of:
+                </p>
+                <ul class="text-body-1">
+                    <li>
+                        The average dengue force of infection (i.e. the per capita annual risk of dengue infection for a
+                        susceptible individual)
+                    </li>
+                    <li>The seroprevalence at 9 years of age</li>
+                    <li>The annual expected number of hospital admissions by age</li>
+                </ul>
+                <p class="text-body-1">
+                    Estimates of these metrics are available at the first and second administrative unit level.
+                </p>
```

Comment:
This now shows up weirdly because of v-container imposing flex layout..

![Screenshot 2024-04-24 103106](https://github.com/mrc-ide/arbomap/assets/44669576/8b67d16d-d19a-489b-b721-d8d165fa5940)

Reply:
Well spotted - on large screens only.